tsconfig.json

```diff
@@ -13,3 +13,4 @@
   },
   "include": ["./*.js", "src/**/*.ts", "test/**/*.ts", "./.eslintrc.cjs"]
 }
+
```

Comment:
This all looks good! But since your controller functions are empty, its gonna cause much less of a headache if I just approve inventory endpoints, routes and controllers all at once, rather than adding the empty methods to main. So I'm gonna close this PR but pls open new one when you have controllers finished